examples/joystick.py

```diff
@@ -71,66 +71,66 @@ def main():
                 # joystick, filling up the list without needing to create them manually.
                 joy = pygame.joystick.Joystick(event.device_index)
                 joysticks[joy.get_instance_id()] = joy
-                print("Joystick {} connencted".format(joy.get_instance_id()))
+                print(f"Joystick {joy.get_instance_id()} connencted")
```

Comment:
Oop, note to self: this is misspelled. Not something that needs to be resolved in this PR though.

Reply:
https://pypi.org/project/codespell is blindingly fast.

Reply:
Good tip, I'll check it out.